examples/market_maker.py

```diff
@@ -0,0 +1,19 @@
+from ucf_exchange_client import Strategy
+
+market_maker = Strategy('mm')
```

Comment:
ah, I thought this was complete. I'm just going to port my old market maker bot to this API.